ra2ce/analyses/indirect/traffic_analysis/equity_analysis.py

```diff
@@ -0,0 +1,100 @@
+"""
+                    GNU GENERAL PUBLIC LICENSE
+                      Version 3, 29 June 2007
+
+    Risk Assessment and Adaptation for Critical Infrastructure (RA2CE).
+    Copyright (C) 2023 Stichting Deltares
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""
+
+import geopandas as gpd
+import pandas as pd
+import numpy as np
+from ra2ce.analyses.indirect.traffic_analysis.accumulated_traffic_dataclass import (
+    AccumulatedTraffic,
+)
+from ra2ce.analyses.indirect.traffic_analysis.traffic_analysis_base import (
+    TrafficAnalysisBase,
+)
+
+
```

Comment:
Compared to the documentation, where are the weights calculated? (simple weights, egalitarianism weight, and prioritarianism weights?